README.md

```diff
@@ -130,28 +130,61 @@ Looking towards the future, our vision is to enhance this platform by integratin
 
 ### Installation
 
-_TODO
-_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
-
-1. Solder your PARTS CRP Board
-2. Solder accessories like the IMU
-3. Assemble the Pololu Romi Chassis
-   ```sh
-   npm install
-   ```
-4. Enter your API in `config.js`
-   ```js
-   const API_KEY = 'ENTER YOUR API';
-   ```
+Welcome to the setup guide for the PARTS Common Robotics Platform. Follow these steps to prepare your hardware and software for an exciting journey into robotics!
+
+#### 1. Solder Your PARTS CRP Board
+Soldering the PARTS CRP board is the first crucial step. Follow the instructions carefully to ensure a successful setup.
+
+- **Interactive BOM**: Utilize the [Interactive BOM](https://htmlpreview.github.io/?https://github.com/portlandrobotics/common_platform/blob/master/hardware/romi_board/bom/ibom.html) for an easier soldering process.
```

Comment:
Maybe this repository can use .github pages to host the bom itself instead of using htmlpreview. Reduction on the use of outside services is always preferable. Pages seems like something that may be around longer if features are every dropped.